### DISCUSSION TITLE
Do we have any velero aws plugin for ppc64le sevrer?

### DISCUSSION THREAD

**Discussion by Monancy** [vmware-tanzu/velero#4632] [Community support Q&A]
When I use "--plugins  velero/velero-plugin-for-aws:v1.3.0" to install a velero, i will receive error message as error: code = Unknown desc = choosing image instance: no image found in manifest list for architecture ppc64le, variant "", OS linux
So do you have any plugins for ppc64le so that i can successfully  install velero? Thanks a lot.

**Comment by reasonerjt** [comment]
Currently not, you'll need to re-compile the code and build your own image.
